pkg/configutil/configutil.go

```diff
@@ -0,0 +1,104 @@
+package configutil
+
+import (
+	"html/template"
+	"os"
+
+	f "github.com/flyteorg/flytectl/pkg/filesystemutils"
+)
+
+const (
+	AdminConfigTemplate = `admin:
+  # For GRPC endpoints you might want to use dns:///flyte.myexample.com
+  endpoint: {{.Host}}
+  authType: Pkce
+  insecure: {{.Insecure}}
+logger:
+  show-source: true
+  level: 0`
+	StorageConfigTemplate = `
+storage:
+  connection:
+    access-key: minio
+    auth-type: accesskey
+    disable-ssl: true
+    endpoint: http://localhost:30084
+    region: us-east-1
+    secret-key: miniostorage
+  type: minio
+  container: "my-s3-bucket"
+  enable-multicontainer: true`
+	StorageS3ConfigTemplate = `
+storage:
+  type: stow	
+  stow:
+    kind: s3
+    config:
+      auth_type: iam
+      region: <replace> # Example: us-east-2
+  container: <replace> # Example my-bucket. Flyte k8s cluster / service account for execution should have read access to this bucket`
+	StorageGCSConfigTemplate = `
+storage:
+  type: stow	
+  stow:
+    kind: s3
+    config:
+      json: ""
+      project_id: <replace-me> # TODO: replace <project-id> with the GCP project ID
+      scopes: https://www.googleapis.com/auth/devstorage.read_write
+  container: <replace> # Example my-bucket. Flyte k8s cluster / service account for execution should have read access to this bucket`
+)
```

Comment:
Can you verify if these actually work.
I have heard from two users who have complained about the aws one which they tried.
```
Error:
1 error(s) decoding:
* '' has invalid keys: config, kind
ERRO[0000]
1 error(s) decoding:
* '' has invalid keys: config, kind  src="main.go:13"
this I got after adding storage in config.yaml
```

Reply:
Lets also update the docs https://docs.flyte.org/projects/flytectl/en/stable/ with right config

Reply:
yes there is no issue with parsing